.github/workflows/automerge.yaml

```diff
@@ -8,23 +8,5 @@ on:
       - 'ci'
 jobs:
   automerge:
-    name: Automerge
-    runs-on: ubuntu-latest
-    if: ${{ github.event.workflow_run.conclusion == 'success' && github.actor == 'nsmbot' }}
-    steps:
-      - name: Check out the code
-        uses: actions/checkout@v2
-      - name: Fetch main
-        run: |
-          git remote -v
-          git fetch --depth=1 origin main
-      - name: Only allow go.mod, go.sum, *.gen.go files
-        run: |
-          find . -type f ! -name 'go.mod' ! -name 'go.sum' -exec git diff --exit-code origin/main -- {} +
-      - name: Merge PR
-        uses: ridedott/merge-me-action@master
-        with:
-          GITHUB_LOGIN: nsmbot
-          ENABLED_FOR_MANUAL_CHANGES: true
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          MAXIMUM_RETRIES: 25
+    if: ${{ github.event.workflow_run.conclusion == 'success' && github.actor == 'nsmbot'}}
+    uses: networkservicemesh/.github/.github/workflows/automerge.yaml@main
```

Comment:
Do we want to depend on a branch here rather than a specific point?

I can see tradeoffs either way... so the real question is 'have we thought it through' ?

Reply:
I see next pros/cons

pros:
- we're always up to date.

cons:
- ci for previous could be unstable.

In my mind the best option is: 
 1. use @main version by default.
 2. change @main version to point on the release. 
 
  think we can address this in the v1.5.0.

Thoughts?

Reply:
Sounds good :)

Reply:
@edwarnicke I've created a ticked for v1.5.0 https://github.com/networkservicemesh/.github/issues/6